src/main/resources/init_db.sql

```diff
@@ -7,3 +7,10 @@ CREATE TABLE `manufacturers` (
                                         `country` VARCHAR(225) NOT NULL,
                                         `is_deleted` TINYINT NOT NULL DEFAULT 0,
                                         PRIMARY KEY (`id`));
+
+CREATE TABLE `drivers` (
+                                        `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+                                        `name` varchar(255) DEFAULT NULL,
```

Comment:
Variable types should be uppercase too

Reply:
Agree with previous comment. please rename in uppercase style